docker-compose.yml

```diff
@@ -28,17 +28,19 @@ services:
     environment:
       - PRODUCT_ID=UNASSIGNED
       - REDIS_ENABLED=false
+      - REDIS_HOST=localhost
```

Comment:
this is required when production == true

Reply:
YAY! No longer have to have a separate changeless on the template just to keep this working locally 😄